src/Tracy/Helpers.php

```diff
@@ -164,10 +164,9 @@ public static function getSource(): string
 			return (!empty($_SERVER['HTTPS']) && strcasecmp($_SERVER['HTTPS'], 'off') ? 'https://' : 'http://')
 				. ($_SERVER['HTTP_HOST'] ?? '')
 				. $_SERVER['REQUEST_URI'];
-		} else {
-			return 'CLI (PID: ' . getmypid() . ')'
-				. ': ' . implode(' ', array_map([self::class, 'escapeArg'], $_SERVER['argv']));
 		}
+		return 'CLI (PID: ' . getmypid() . ')'
```

Comment:
why?

Reply:
Suggestion only, but it can be converted into if-branch (clearer intention, lower complexity numbers).

Reply:
if it's not related the phpstan error it should not be part of this PR

Reply:
Reverted.